SpeakerImage.cs

```diff
@@ -12,7 +12,7 @@ public class SpeakerImage : SyncModule
                 ? input
                     .Clone(new MetadataItems
                     {
-                        { SiteKeys.Image, input.ContainsKey(SiteKeys.GitHub) ? input.GetString(SiteKeys.GitHub) + ".png" : "/img/dot_bot.png" }
+                        { SiteKeys.Image, input.ContainsKey(SiteKeys.GitHub) ? input.GetString(SiteKeys.GitHub).TrimEnd('/') + ".png" : "/img/dot_bot.png" }
```

Comment:
At least one of the new speaker files included a trailing slash for the GitHub URL - that doesn't hurt the link or any API integration, but the special `.png` extension trick doesn't like it so we've got to strip it when figuring out the image.